device/kokkos/include/traccc/kokkos/seeding/seeding_algorithm.hpp

```diff
@@ -0,0 +1,56 @@
+/** TRACCC library, part of the ACTS project (R&D line)
```

Comment:
Let's not put this class into this PR at all. Without the `seed_finding` class existing it can't do anything yet anyway. You should instead instantiate `traccc::kokkos::spacepoint_binning` itself in the example code.